### PR TITLE
Document migration process

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Fixed tickets | 
| License       | MIT

### CODE REVIEW

File: docs/migration.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
+Migration from IvoryCKEditorBundle to FOSCKEditorBundle
+=======================================================
+
+Here we will explain process of migration.
```
Comment (greg0ire) [inline docs/migration.rst]: "the process"

File: docs/migration.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+Update composer.json
+--------------------
+
+Replace:
+
+.. code-block:: json
+
+    {
+     "require": {
```
Comment (greg0ire) [inline docs/migration.rst]: Broken indention

File: docs/migration.rst [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        "egeloen/ckeditor-bundle": "*"
+        }
+    }
+
+With:
+
+.. code-block:: json
+
+    {
+     "require": {
```
Comment (greg0ire) [inline docs/migration.rst]: Here too

File: docs/migration.rst [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+            ];
+
+            // ...
+        }
+    }
+
+Update configuration root key
+------------------------------
+
+Only the root key of configuration is changed.
```
Comment (greg0ire) [inline docs/migration.rst]: "the configuration"

File: docs/migration.rst [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    fos_ck_editor:
+        configs:
+            my_config:
+                toolbar: [ ["Source", "-", "Save"], "/", ["Anchor"], "/", ["Maximize"] ]
+                uiColor:                "#000000"
+                filebrowserUploadRoute: "my_route"
+                extraPlugins:           "wordcount"
+                # ...
+
+If you are not using Symfony Flex, then replace root key in ``app/config/config.yml``.
```
Comment (greg0ire) [inline docs/migration.rst]: "the root key"

File: docs/migration.rst [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+----------------
+
+The main thing that changed is the namespace, so you will have to find
+all occurrences of ``Ivory\CKEditorBundle\*`` in your application and
+replace them with ``FOS\CKEditorBundle\*``.
+
+Update service definition
+-------------------------
+
+If you are fetching any of the services directly for container you will
```
Comment (greg0ire) [inline docs/migration.rst]: "from the container"

File: docs/migration.rst [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+all occurrences of ``Ivory\CKEditorBundle\*`` in your application and
+replace them with ``FOS\CKEditorBundle\*``.
+
+Update service definition
+-------------------------
+
+If you are fetching any of the services directly from the container you
+will have to find all occurrences of ``ivory_ck_editor.*`` in your application
+and replace them with ``fos_ck_editor.*``.
+
```
Comment (ElectricMaxxx) [inline docs/migration.rst]: i would add a form type example here as this is the major usecase.

Comment (kunicmarko20) [reply docs/migration.rst]: Like: instead of `ivory_ck_editor.form.type` you fetch `fos_ck_editor.form.type`?

**Comment by ElectricMaxxx** [comment]
I would also add some lines, maybe in a hint section of index, to tell the history, why a migration is neede, and why the docs look differnt now.

File: docs/migration.rst (hunk 7 above)
Comment (ElectricMaxxx) [reply docs/migration.rst]: not really, but i think i should have done my note some lines above among the namespaces: I mean Something like:
```
$form->add('body',  CKEditorType::Class)
```

Comment (ElectricMaxxx) [reply docs/migration.rst]: for sure including use definitions

Comment (kunicmarko20) [reply docs/migration.rst]: Well, that stays the same, the only thing that would change is the `use` statement?

**Comment by kunicmarko20** [comment]
Added more examples and added mention of migration in the Readme instead of index.

**Comment by kunicmarko20** [comment]
As only you and I are active, can we merge this?

**Pull request merged**